### PR TITLE
MULTI.TRA to chinese

### PR DESCRIPTION
**Pull request by Lzw104522773**
MULTI.TRA to chinese

### CODE REVIEW

**Comment by skellytz** [comment]
Last time I tested the `WEIDU_OS` variable for setup prompts, Project Infinity couldn't properly understand it. I don't think that issue has been solved. Seeing as _setup.tra_ is only for WeiDU prompts and is excluded from character conversion, simply make it UTF-8 and don't split it into OS-specific files. For now we'll just have to assume Chinese players use Console or Terminal with UTF-8 support.

File: arestorationp/arestorationp.tp2 [hunk 1]
```diff
@@ -94,9 +94,8 @@ LANGUAGE ~Foeroyskr (BG2EE) (by inscrutable horse)~
 LANGUAGE ~Simplified Chinese (by Sebastian C)~
 		 ~schinese~
          ~arestorationp/language/english/game.tra~
-         ~arestorationp/language/english/setup.tra~
-         ~arestorationp/language/schinese/game.tra~
```
Comment (skellytz) [inline arestorationp/arestorationp.tp2:98]: Keep this. US-ASCII is part of UTF-8, so if any new untranslated strings are added, they'll work for Chinese players, too.

**Comment by Lzw104522773** [comment]
OK！

**Comment by skellytz** [comment]
Thanks!

**Pull request merged**